src/utils/rounded.jsx

```diff
@@ -0,0 +1,21 @@
+"use strict";
+
+export default function rounded(x, y, w, h, cornerRadius) {
+  console.log(cornerRadius)
+  var retval;
+  retval  = "M" + (x + cornerRadius.topLeft) + "," + y;
+  retval += "h" + (w - cornerRadius.topLeft - cornerRadius.topRight);
+  if (cornerRadius.topRight != 0) { retval += "a" + cornerRadius.topRight + "," + cornerRadius.topRight + " 0 0 1 " + cornerRadius.topRight + "," + cornerRadius.topRight; }
```

Comment:
Or, you could go totally crazy and do it like this using template literals (not tested or anything, just a naive code copy and edit):
```javascript
export default function rounded(x, y, w, h, cornerRadius) {
  var retval;

  const { topLeft, topRight, bottomLeft, bottomRight } = cornerRadius; 
  
  retval  = `M${x + topLeft},${y}h${w - topLeft - topRight}`;
  
  if (topRight === 0) { 
    retval += `h${topRight}v${topRight}`;
  } else {
    retval += `a${topRight},${topRight} 0 0 1 ${topRight},${topRight}`;
  }
    
  retval += `v${h - topRight - bottomRight}`;
  
  if (bottomRight === 0) {
    retval += `v${bottomRight}h${-bottomRight}`;
  } else {
    retval += `a${bottomRight},${bottomRight} 0 0 1 ${-bottomRight},${bottomRight}`;
  }

  retval += `h${bottomLeft + bottomRight - w}`;

  if (bottomLeft === 0) {
    retval += `h${-bottomLeft}v${-bottomLeft}`;
  } else {
    retval += `a${bottomLeft},${bottomLeft} 0 0 1 ${-bottomLeft},${-bottomLeft}`;
  }

  retval += `v${topLeft + bottomLeft - h}`;

  if (topLeft === 0) {
    retval += `v${-topLeft}h${topLeft}`;
  } else {
    retval += `a${topLeft},${topLeft} 0 0 1 ${topLeft},${-topLeft}`;
  }

  retval += "z";
  return retval;
}
```
...but that's not necessarily more legible, now that I look at it. But definitely do the destructuring assignment